### PR TITLE
Added JSON-LD file + thumbnail for SPARC

### PR DESCRIPTION
**Pull request by andreasbueckle**
Performed the following:

- created sparc folder on same level as kpmp folder
- added JSON-LD file for SPARC (26 registrations)
- added thumbnail

### CODE REVIEW

File: projects/ccf-eui/src/assets/sparc/data/rui_locations.jsonld [hunk 1]
```diff
@@ -0,0 +1,1963 @@
...
+      "@id": "has_ontology_term",
+      "@type": "@id"
+    },
+    "thumbnail": {
+      "@id": "has_thumbnail"
+    }
+  },
+  "@graph": [
+    {
+      "@id": "https://doi.org/10.1161/CIRCRESAHA.120.318458",
```
Comment (bherr2) [inline projects/ccf-eui/src/assets/sparc/data/rui_locations.jsonld:38]: All `@id`s must be unique. For these, just add a #Donor1, #TissueBlock1, etc to the end of the url like this:
`https://doi.org/10.1161/CIRCRESAHA.120.318458#Donor1`

File: projects/ccf-eui/src/assets/sparc/data/rui_locations.jsonld [hunk 2]
```diff
@@ -0,0 +1,1963 @@
...
+      "label": "Apical Septum Female",
+      "description": "Entered 3/16/2021, Peter Hanna, SPARC/UCLA",
+      "d": "Exemplary tissue registration for paper titled \"Innervation and Neuronal Control of the Mammalian Sinoatrial Node: A Comprehensive Atlas\"",
+      "link": "https://doi.org/10.1161/CIRCRESAHA.120.318458",
+      "consortium_name": "SPARC",
+      "provider_name": "SPARC/UCLA",
+      "provider_uuid": "TBD",
+      "samples": [
+        {
+          "@id": "https://doi.org/10.1161/CIRCRESAHA.120.318458",
```
Comment (bherr2) [inline projects/ccf-eui/src/assets/sparc/data/rui_locations.jsonld:49]: `https://doi.org/10.1161/CIRCRESAHA.120.318458#TissueBlock1`

File: projects/ccf-eui/src/assets/sparc/data/rui_locations.jsonld [hunk 3]
```diff
@@ -0,0 +1,1963 @@
...
+          "label": "Registered 3/16/2021, Peter Hanna, SPARC/UCLA",
+          "description": "Exemplary tissue registration for paper titled \"Innervation and Neuronal Control of the Mammalian Sinoatrial Node: A Comprehensive Atlas\"",
+          "link": "https://doi.org/10.1161/CIRCRESAHA.120.318458",
+          "sample_type": "Tissue Block",
+          "section_count": "",
+          "section_size": "",
+          "section_units": "",
+          "rui_location": {
+            "@context": "https://hubmapconsortium.github.io/hubmap-ontology/ccf-context.jsonld",
+            "@id": "http://purl.org/ccf/1.5/36d378f4-d4b5-456f-a32c-12a4c60d8159",
```
Comment (bherr2) [inline projects/ccf-eui/src/assets/sparc/data/rui_locations.jsonld:60]: This one is always unique as it comes from the RUI, no need to change.

File: projects/ccf-eui/src/assets/sparc/data/rui_locations.jsonld [hunk 4]
```diff
@@ -0,0 +1,1963 @@
...
+              "link": "https://doi.org/10.1161/CIRCRESAHA.120.318458",
+              "technology": "OTHER",
+              "thumbnail": "assets/sparc/thumbnails/sparc_logo.jpg"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "@id": "https://doi.org/10.1161/CIRCRESAHA.120.318458",
```
Comment (bherr2) [inline projects/ccf-eui/src/assets/sparc/data/rui_locations.jsonld:113]: `https://doi.org/10.1161/CIRCRESAHA.120.318458#Donor2` .. and so on

File: projects/ccf-eui/src/assets/sparc/data/rui_locations.jsonld [hunk 5]
```diff
@@ -0,0 +1,1963 @@
...
+              "rotation_units": "degree",
+              "x_translation": 101.7,
+              "y_translation": 22.763,
+              "z_translation": 61.069,
+              "translation_units": "millimeter"
+            }
+          },
+          "datasets": [
+            {
+              "@id": "https://doi.org/10.1161/CIRCRESAHA.120.318458",
```
Comment (bherr2) [inline projects/ccf-eui/src/assets/sparc/data/rui_locations.jsonld:100]: `https://doi.org/10.1161/CIRCRESAHA.120.318458#Dataset1`

**Review by bherr2 — CHANGES_REQUESTED** [review]
@andreasbueckle IDs need to be fixed, but otherwise looks good! Let me know if you need any help with this.

**Comment by andreasbueckle** [comment]
@bherr2 
I performed the changes you requested and committed the changes:
https://github.com/hubmapconsortium/ccf-ui/commit/c172fbeda3563df32f1e10da5de1248bf1767b4c

File: projects/ccf-eui/src/assets/sparc/data/rui_locations.jsonld (hunk 1 above)
Comment (andreasbueckle) [reply projects/ccf-eui/src/assets/sparc/data/rui_locations.jsonld]: Thanks, done.

**Review by bherr2 — APPROVED** [review]

**Comment by andreasbueckle** [comment]
@bherr2 
Adjusted dimensions of sparc thumbnail to 800 x 600

**Pull request merged**